### DISCUSSION TITLE
Reduced column size for the description

### DISCUSSION THREAD

**Discussion by brlarini** [dani-garcia/vaultwarden#3709] [Q&A]
Hi everyone.
I was some versions behind, and after updating to the current one (1.29.0), I noticed the column reserved for the description of the credentials was reduced, while there is a pretty thick blank space to the right.
Is it some layout bug? Is there some way to expand the column so we can read more of the items description?
I'm on Firefox 115.

Thanks!

![image](https://github.com/dani-garcia/vaultwarden/assets/23079160/1980984f-a7bd-46af-bc01-697989681a63)

**Comment by BlackDex** [comment]
Please try out in a private tab/window. Or try to clear your browser cache.

**Reply by brlarini** [reply]
Hi, thanks for replying!
I've tried opening in a private window, in another browser in which I've never used Vaultwarden and in another computer but the behavior persists in any of those.
I've also tried restarting nginx. It isn't caching but acts as a reverse proxy to Vaultwarden.

I should've told this before, but it is running on a Docker container.

Any ideas?

**Reply by tessus** [reply]
@BlackDex I can see this as well. As you can see half of the vault data column is just empty.

However, I doubt that this is a vw issue, but rather the web-vault itself. Another interesting design choice by the bw devs. ;-)

![image](https://github.com/dani-garcia/vaultwarden/assets/223439/7ca943a8-a2e7-488f-9205-beb3f9c18944)
